README.md

```diff
@@ -25,36 +25,72 @@ Please always double check the version of Keptn you are using compared to the ve
+## Setup Prometheus Monitoring
+
+Keptn doesn't install or manage Prometheus and its components. Users need to install Prometheus and Prometheus Alert manager as a prerequisite.
+
+Some environment variables have to set up in the prometheus-service deployment
+```yaml
+    # Prometheus installed namespace
+    - name: PROMETHEUS_NS
+      value: 'default'
+    # Prometheus server configmap name
+    - name: PROMETHEUS_CM
+      value: 'prometheus-server'
+    # Prometheus server app labels
+    - name: PROMETHEUS_LABELS
+      value: 'component=server'
+    # Prometheus configmap data's config filename
+    - name: PROMETHEUS_CONFIG_FILENAME
+      value: 'prometheus.yml'
+    # AlertManager configmap data's config filename
+    - name: ALERT_MANAGER_CONFIG_FILENAME
+      value: 'alertmanager.yml'
+    # Alert Manager config map name
+    - name: ALERT_MANAGER_CM
+      value: 'prometheus-alertmanager'
+    # Alert Manager app labels
+    - name: ALERT_MANAGER_LABELS
+      value: 'component=alertmanager'
+    # Alert Manager installed namespace
+    - name: ALERT_MANAGER_NS
+      value: 'default'
+    # Alert Manager template configmap name
+    - name: ALERT_MANAGER_TEMPLATE_CM
+      value: 'alertmanager-templates'
 ```
 
-## Deploy in your Kubernetes cluster
+### Execute the following steps to install prometheus-service
 
-To deploy the current version of the *prometheus-service* in your Keptn Kubernetes cluster, use the `deploy/*.yaml` files from this repository and apply it:
+* Download the Keptn's Prometheus service manifest
+```bash
+wget https://raw.githubusercontent.com/keptn-contrib/prometheus-service/release-0.4.0/deploy/service.yaml
+```
 
-```console
-kubectl apply -f deploy/service.yaml
-kubectl apply -f deploy/distributor.yaml
+* Replace the environment variable value according to the use case and apply the manifest
+```bash
+kubectl apply -f service.yaml
 ```
 
-## Delete in your Kubernetes cluster
+* Install Role and Rolebinding to permit Keptn's prometheus-service for performing operations in the Prometheus installed namespace.
+```bash
+kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/release-0.4.0/deploy/role.yaml -n <PROMETHEUS_NS>
+```
+
+* Execute the following command to install Prometheus and set up the rules for the *Prometheus Alerting Manager*:
+```bash
+keptn configure monitoring prometheus --project=sockshop --service=carts
+```
 
-To delete a deployed *prometheus-service*, use the `deploy/*.yaml` files from this repository and delete the Kubernetes resources:
+### Optional: Verify Prometheus setup in your cluster
 
-```console
-kubectl delete -f deploy/distributor.yaml
-kubectl delete -f deploy/service.yaml
+* To verify that the Prometheus scrape jobs are correctly set up, you can access Prometheus by enabling port-forwarding for the prometheus-service:
+```bash
+kubectl port-forward svc/prometheus-service 8080 -n <PROMETHEUS_NS>
```

Comment:
should this forward to the Prometheus installation or the Prometheus service? 
The prometheus-service would rather be in the keptn namespace but it won't expose the prometheus installation

Reply:
Done